resources/language/resource.language.en_gb/strings.po

```diff
@@ -811,3 +811,35 @@ msgstr ""
 msgctxt "#30198"
 msgid "Exporting new episodes"
 msgstr ""
+
+msgctxt "#30199"
+msgid "Shared library (Need Kodi MySql server database)"
+msgstr ""
+
+msgctxt "#30200"
+msgid "Use MySQL shared library database"
+msgstr ""
+
+msgctxt "#30201"
+msgid "Username"
+msgstr ""
+
+msgctxt "#30202"
```

Comment:
Duplicated string, same as #30004

Reply:
little pain, it is not worth wasting time to change all the commits and the code for a string
in case if necessary we can reuse it for something else